WebIDL/ecmascript-binding/constructors.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Realm for constructed objects</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id=log></div>
+<script>
+function object_realm(dp) {
+  var url = DOMParser.prototype.parseFromString.call(dp, "x", "text/html").documentURI;
+  var file = url.slice(url.lastIndexOf("/") + 1);
```

Comment:
It's really unclear why the documentURI would be derived from the this value, looking at the DOMParser spec :(. Maybe add a comment pointing to the spec, or if this is underspecified, open an issue and then add a comment pointing to the issue?